Bundler/Viewer/Unity/Assets/Scripts/Network/Scripts/NetworkAnchorManager.cs

```diff
@@ -0,0 +1,487 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEngine.Networking;
+using UnityEngine.XR.WSA;
```

Comment:
should this be inside the UNITY_WSA check?

Reply:
"won't fix".  Wasn't needed.  Win32 and UWP built fine with namespace.  Leaving as is